container-images/kolla/base/uid_gid_manage.sh

```diff
@@ -65,6 +65,7 @@ _SUPPORTED_USERS['ovn-bgp']='ovn-bgp 42486 42486 /var/lib/ovn-bgp kolla'
 _SUPPORTED_USERS['placement']='placement 42482 42482 /var/lib/placement kolla'
 _SUPPORTED_USERS['qemu']='qemu 107 107'
 _SUPPORTED_USERS['rabbitmq']='rabbitmq 42439 42439 /var/lib/rabbitmq kolla'
+_SUPPORTED_USERS['rally']='rally 42480 42480 /var/lib/rally kolla'
```

Comment:
It is the userid and groupid which we need to specify while creating the user with in the container. It has a sequence starting with 424. I am not sure about last two digits. It might be random and unique. May be we can use the random and different id for rally.

Reply:
From background perspective it used to be coming from kolla side and there were some constraints related to rpm as well but i don't recall. 

Although it's dropped in kolla but i think can use that value here and keep seperate from tempest:- https://github.com/openstack/kolla/blob/428f37124986019bedc897469f9b1f3470b62082/kolla/common/users.py#L160-L162